specification/storagesync/resource-manager/Microsoft.StorageSync/2017-06-05-preview/storagesync.json

```diff
@@ -0,0 +1,3232 @@
+          },
+          {
+            "name": "syncGroupName",
+            "in": "path",
+            "description": "Name of Sync Group resource.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "ServerEndpoint Resource"
+        ],
+        "operationId": "ServerEndpointsGet_ListBySyncGroup",
+        "description": "Get a ServerEndpoint list.",
+        "x-ms-examples": {
+          "ServerEndpointsGet_ListBySyncGroup": {
+            "$ref": "./examples/ServerEndpointsGet_ListBySyncGroup.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Array of Server Endpoint resources in Sync Group",
+            "schema": {
+              "$ref": "#/definitions/ServerEndpointArray"
+            },
+            "headers": {
+              "Location": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              },
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error message indicating why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/storageSyncServices/{storageSyncServiceName}/syncGroups/{syncGroupName}/serverEndpoints/{serverEndpointName}/recallAction": {
+      "post": {
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "storageSyncServiceName",
+            "in": "path",
+            "description": "Name of Storage Sync Service resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "syncGroupName",
+            "in": "path",
+            "description": "Name of Sync Group resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serverEndpointName",
+            "in": "path",
+            "description": "Name of Server Endpoint object.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "ServerEndpoint Resource",
+          "Actions"
+        ],
+        "operationId": "ServerEndpointRecall_Post",

```

Comment:
Operation ID should be in the format `"group_verb"`. So this should be changed to something like:
`"operationId": "ServerEndpoints_Recall"`

Reply:
resolved